CMakeLists.txt

```diff
@@ -297,12 +297,16 @@ ELSE()
       OPTION(PA_USE_ALSA "Enable support for ALSA" OFF)
     ENDIF()
     IF(PA_USE_ALSA)
+      OPTION(PA_ALSA_IGNORE_ALL_PLUGINS "Ignore all ALSA plugin devices" OFF)
```

Comment:
I notice that several ALSA options are controlled by environment variables. See multiple calls to getenv() in "pa_linux_alsa.c". That would allow a user to disable plugin devices for a precompiled app. If the plugin devices are specific to a machine then using getenv() might be better than disabling plugins for the app on all machines.